### ISSUE TITLE
maestro samples run hello-k8s failed

### ISSUE THREAD

**Issue by msftcoderdjw** [eclipse-symphony/symphony#72]
<img width="566" alt="image" src="https://github.com/eclipse-symphony/symphony/assets/59427055/40faef73-8748-4b28-8185-40b508babd65">

**Comment by Haishi2016** [comment]
Are you running this against minikube? It will have problem allocating a public IP for the service. Maybe we have to customize the templates for minikube.

**Comment by msftcoderdjw** [comment]
Yes I tried this against minikube but I have started tunnel. I founnd this seems to be postAction script is executed too quick, at that moment, service "sample-prometheus-instance" was not started yet. I modified meastro a bit and add verbose log,

<img width="809" alt="image" src="https://github.com/eclipse-symphony/symphony/assets/59427055/4a4d466a-91d1-4f39-8257-21e7af3cbbdf">

After waiting a couple of seconds, we can see the postAction script can run successfuly.
<img width="1295" alt="image" src="https://github.com/eclipse-symphony/symphony/assets/59427055/965199b1-594b-420e-8ef5-da3dbafa70ad">
